Makefile

```diff
@@ -13,6 +13,7 @@ tmp/cache-image.tar:
 	docker save -o tmp/cache-image.tar cache-image
 
 .PHONY: lint
+lint:
```

Comment:
I think such changes could be extracted and pushed directly to matser.

Reply:
I'm not sure that removing `luacheck` should be in a master. I've broke Makefile in the `completely-rewrite-in-go`.

Reply:
I don't see that you remove "luacheck" here

Reply:
I've removed it in the patch that broke Makefile.